src/collectd.conf.pod

```diff
@@ -8148,6 +8148,41 @@ want to use authentication all three fields must be set.
 
 =back
 
+=head2 Plugin C<write_prometheus>
+
+The I<write_prometheus plugin> implements a tiny webserver that can be scraped
+using I<Prometheus>.
+
+B<Options:>
+
+=over 4
+
+=item B<Port> I<Port>
+
+Port the embedded webserver should listen on. Defaults to B<9103>.
+
+=item B<StalenessDelta> I<Seconds>
```

Comment:
I think this setting is going to cause weirdness. If you're going to expose timestamps, always expose timestamps for a use case like this.

This is also what you will be doing once we fix staleness handling.